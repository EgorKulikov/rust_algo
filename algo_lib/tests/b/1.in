2 1 3 5