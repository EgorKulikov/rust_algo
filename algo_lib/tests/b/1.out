0.5667292175235064
